### ISSUE TITLE
Scaling up Matter.Image doesn't resize body correctly

### ISSUE THREAD

**Issue by goldfire** [photonstorm/phaser#3951]
## Version

* Phaser Version: 3.12.0-beta2

* Operating system: macOS 10.13.5

* Browser: Chrome

## Description

If you scale a sprite with a Matter.js body on it down and then scale it back up, the body will never get larger again. The easiest way to explain this is to just look at the demo:

https://codepen.io/goldfire/pen/wEwoyd

Both of the bodies should be the exactly same size since both sprites are scaled to 1.2, but the first one is small because it was first scaled to 0.1 before being scaled back up.

**Comment by photonstorm** [comment]
Sounds to me like #3785 ?

**Comment by photonstorm** [comment]
To be honest it’s a pain in the arse,  it I can’t see an easy solution for it.

**Comment by bergben** [comment]
I think you could just track the scaling factor manually and apply the fix as I posted in #3785 for whenever scaling is used with the matter.js physics engine? @photonstorm

**Comment by photonstorm** [comment]
@bergben yeah I figured it was worth adding just to keep it consistent, so it's now in the master branch.
